### DISCUSSION TITLE
Booleans for silent part list export

### DISCUSSION THREAD

**Discussion by ancelin-charpentier** [cwapi3d/cwapi3dpython#229] [Q&A] [answered]
Hey there,

I am using the following command but I got no clue about what is supposed to be set for a1, a2 and a3.
I guess a1 is the indentation maybe? I set "1" and it seems to be happy...
how about a2 and a3? 

`def generate_new_part_list_silently(a0: List[int], a1: int, a2: bool, a3: bool) -> None:`

    """generate new part list silently

    Parameters:
        a0: a0
        a1: a1
        a2: a2
        a3: a3

    Returns:
        None
    """

Cheers

**Comment by Brunner246** [comment, marked as answer]
Hi @ancelin-charpentier 

Sorry, this is a function that was unfortunately skipped by our automatism for the descriptions. 

please have a look at the c++ interface. 
https://github.com/cwapi3d/cwapi3dcpp/blob/main/include/cwapi3d/ICwAPI3DListController.h#L82

**Reply by ancelin-charpentier** [reply]
Noted, thanks.
